encord/collection.py

```diff
@@ -0,0 +1,117 @@
+import uuid
+from typing import List, Optional
+from uuid import UUID
+
+from encord.exceptions import (
+    AuthorisationError,
+)
+from encord.http.v2.api_client import ApiClient
+from encord.orm.collection import Collection as OrmCollection
+from encord.orm.collection import (
+    CreateCollectionParams,
+    CreateCollectionPayload,
+    GetCollectionParams,
+    GetCollectionsResponse,
+    UpdateCollectionPayload,
+)
+
+
+class Collection:
+    """
+    Represents collections in Index.
+    Collections are a logical grouping of data items that can be used to
+    create datasets and perform various data curation flows.
+    """
+
+    def __init__(self, client: ApiClient, orm_collection: OrmCollection):
+        self._client = client
+        self._collection_instance = orm_collection
+
+    @property
+    def collection_hash(self) -> uuid.UUID:
```

Comment:
[nitpick]💅

This is better IMO

```suggestion
    def uuid(self) -> uuid.UUID:
```

Reply:
also change other references of `collection_hash` to `uuid` and the docstrings as well